cluster/examples/kubernetes/cassandra/operator.yaml

```diff
@@ -0,0 +1,227 @@
+
+# Namespace where Cassandra Operator will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rook-cassandra-system
+
+---
+
+# Cassandra Cluster CRD
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: clusters.cassandra.rook.io
+spec:
+  group: cassandra.rook.io
+  names:
+    kind: Cluster
+    listKind: ClusterList
+    plural: clusters
+    singular: cluster
+  scope: Namespaced
+  version: v1alpha1
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          type: object
+          properties:
+            version:
+              type: string
+              # Only allow 3.x.x versions
+              pattern: ^3(\.[0-9]{1,2}){0,2}$
+              description: "Version of Cassandra"
+            datacenter:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: "Datacenter Name"
+                racks:
+                  type: array
+                  properties:
+                    name:
+                      type: string
+                    members:
+                      type: integer
+                    configMapName:
+                      type: string
+                    storage:
+                      type: object
+                      properties:
+                        volumeClaimTemplates:
+                          type: object
+                          # TODO: Check if we can ref the already existing schema
+                      required:
+                       - "volumeClaimTemplates"
+                    placement:
+                      type: object
+                    resources:
+                      type: object
+                      properties:
+                        # TODO: Check if we can ref the already existing schema
+                        cassandra:
+                          type: object
+                        sidecar:
+                          type: object
+                      required:
+                        - "cassandra"
+                        - "sidecar"
+                    sidecarImage:
+                      type: object
+                  required:
+                    - "name"
+                    - "members"
+                    - "storage"
+                    - "resources"
+              required:
+                - "name"
+          required:
+            - "version"
+            - "datacenter"
+
+---
+
+## Cassandra Member CRD
+#apiVersion: apiextensions.k8s.io/v1beta1
+#kind: CustomResourceDefinition
+#metadata:
+#  name: members.cassandra.rook.io
```

Comment:
is this members kind intended to be included in this PR, or will it stay commented out?

Reply:
Originally, the plan was to create a Member CRD for each Cassandra instance.
After some consideration, I decided it would be better to start off using labels on the ClusterIP Services that each instance uses as a static IP. In other words, defer the extra complexity for the future, because it is likely that we will never need it. This is also captured in the `sidecar.md` design doc of #1931.

That being said, this comment is confusing and unnecessary and will be removed.